lib/_emerge/main.py

```diff
@@ -573,6 +574,10 @@ def parse_opts(tmpcmdline, silent=False):
             + "Emerge will ignore matching binary packages. ",
             "action": "append",
         },
+        "--onlydeps-with-ideps": {
+            "help": "modify interpretation of depedencies to include IDEPEND",
+            "choices": true_y_or_n,
+        },
         "--onlydeps-with-rdeps": {
             "help": "modify interpretation of depedencies",
```

Comment:
```suggestion
            "help": "modify interpretation of dependencies",
```

Reply:
This one caught a few others that probably came from the same source and have been copy/pasted around.